protoc-gen-swagger/genswagger/types.go

```diff
@@ -105,7 +105,7 @@ type swaggerOperationObject struct {
 	Tags        []string                `json:"tags,omitempty"`
 	Deprecated  bool                    `json:"deprecated,omitempty"`
 
-	Security     []swaggerSecurityRequirementObject  `json:"security,omitempty"`
+	Security     *[]swaggerSecurityRequirementObject `json:"security,omitempty"`
```

Comment:
Is this required? Can't we just have an empty slice?

Reply:
Required. The `omitempt` tag removes an empty slice. However we don't want to print `null` value in every JSON response so we can't remove this `omitempty` tag.

Reply:
OK